app/src/main/java/org/jellyfin/androidtv/details/actions/PlayFromBeginningAction.kt

```diff
@@ -0,0 +1,23 @@
+package org.jellyfin.androidtv.details.actions
+
+import android.content.Context
+import android.util.Log
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.GlobalScope
+import kotlinx.coroutines.launch
+import org.jellyfin.androidtv.R
+
+private const val LOG_TAG = "PlayFromBeginningAction"
+
+class PlayFromBeginningAction(context: Context, val itemID: String) : PlaybackAction(ActionID.PLAY_FROM_BEGINNING.id, context) {
+	init {
+	    label1 = context.getString(R.string.lbl_play)
```

Comment:
Why is this called `label` is that a google thingy or our thingy?

Reply:
That's a [Google thing](https://developer.android.com/reference/androidx/leanback/widget/Action) sadly, I dislike the name as well.

Reply:
The should've called it "title" and "subtitle" or something. We can't do anything about it though